Islandora-Devops/isle-dc
# Reinstall islandora & views for make local

This may need some refinement but after a lot of testing I notices disabling then re-enabling the views with a module reinstall fixed the issue. It seems to have fix the issues described in https://github.com/Islandora-Devops/isle-dc/issues/212
and https://github.com/Islandora-Devops/isle-dc/issues/136
## What changed
Added a make command to fix issue and a custom script that will only fix the issue if it's actually the issue.

## To test (`make local` only) 
Assuming you've ran `make clean` followed by `make local` but not have triggered the error yet. If you have either run the previous steps (to check the skip works) or just skip to step #4.
1. Run `make fix_views` and it will try to copy the file into the container, run it and remove the script. 
2. Try making creating a basic page
3. Should see error message
4. Run steps 1 & 2 again
5. No more error message

## What just happened
Before attempting to make a page there was no error triggered. In the shell script is a check to look for a specific error. This way the fix is only applied when it absolutely will address the issue.
- It installs Drupal Console
- It runs Drupal console commands to re-enable all of the views one by one
- Installs devel 
- Runs Devel's reinstall on the Islandora module
- Rebuilds caches and node access 
- Runs drush cron
- Then removes the patch_views.sh shell file from container. 

## Other issues addressed
Wasn't sure the impact of the missing config/sync directory and the masonry errors. So to eliminate those as compounding factors I addressed them in this PR as well as the "flysystem module is missing" errors. Some part of the build process is expecting it.
